docs/dotnet/dotnet-signalr-metrics.md

```diff
@@ -0,0 +1,68 @@
+# Semantic Conventions for SignalR metrics emitted by ASP.NET Core
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions for SignalR metrics emitted by .NET components and runtime.
+
+**Disclaimer:** These are initial .NET metric instruments available in .NET 8 but more may be added in the future.
+
+<!-- toc -->
+
+- [Metric: `signalr_http_transport.connection.duration`](#metric-signalr_http_transportconnectionduration)
+- [Metric: `signalr_http_transport.active_connections`](#metric-signalr_http_transportactive_connections)
+
+<!-- tocstop -->
+
+## Metric: `signalr_http_transport.connection.duration`
+
+**Status**: [Experimental][DocumentStatus]
+
+This metric is required.
+
+<!-- semconv metric.signalr.server.connection.duration(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `signalr_http_transport.connection.duration` | Histogram | `s` | The duration of connections on the server. [1] |
+
+**[1]:** **TODO: why http_transport? can we report protocol name as dimension and not a metric name?**
+**TODO: we should do signalr.server**
+Corresponding `EventCounter` name is `connections-duration`; Meter name is `Microsoft.AspNetCore.Http.Connections`
+<!-- endsemconv -->
+
+<!-- semconv metric.signalr.server.connection.duration(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `signalr.http_transport.status` | string | TODO [1] | `TODO` | Recommended |
+| `signalr.http_transport.transport` | string | TODO [2] | `TODO` | Recommended |
+
+**[1]:** **TODO: why not signalr.status? What's HTTP-specific about it**
+
+**[2]:** **TODO: why not network.transport?**
+<!-- endsemconv -->
+
+## Metric: `signalr_http_transport.active_connections`
+
+**Status**: [Experimental][DocumentStatus]
+
+This metric is required.
+
+<!-- semconv metric.signalr.server.active_connections(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `signalr_http_transport.active_connections` | Histogram | `s` | Number of connections that are currently active on the server. [1] |
+
+**[1]:** **TODO: why http_transport? can we report protocol as dimension and not a metric name?**
+**TODO: we should do signalr.server**
+Corresponding `EventCounter` name is `current-connections`; Meter name is `Microsoft.AspNetCore.Http.Connections`
+<!-- endsemconv -->
+
+<!-- semconv metric.signalr.server.active_connections(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `signalr.http_transport.status` | string | TODO [1] | `TODO` | Recommended |
```

Comment:
Porting active comment threads from the word doc:
NOTE: In the original word doc this attribute being commented on was signalr_http_transport.transport. I'm not sure if this was an intentional change or a transciption error.

## Liudmila Molkova
network.transport

July 18, 2023 at 4:43 PM
## James Newton-King
This value can be long polling, web sockets or server sent events. Does network.transport support those values?


If not, I suggest signalr_http_transport.status and signalr_http_transport.transport

July 18, 2023 at 7:50 PM
## Noah Falk
I changed it, same as above

July 19, 2023 at 4:49 AM
## Liudmila Molkova
It seems these signalr metrics are about http only - if it's the case, why not use http.response.status_code - are there any non-http status signalr will return?thinking about websockets non-http case, we'd probably want different set of status codes and different attribute names anyway - no need to merge HTTP and non-HTTP. Thoughts? 

July 21, 2023 at 6:50 PM